assets/components/atoms/drawer/drawer.js

```diff
@@ -0,0 +1,30 @@
+/* global $ */
+
+const drawer = () => {
+  $('.drawer-toggle').click((e) => {
+    e.preventDefault();
+    const $drawer = $(e.currentTarget).parent('.drawer');
```

Comment:
Favour jQuery selector `$(this)` instead of `e.currentTarget`. It's the only case where arrow function are not allowed 😉